Offices/Offices.API/Program.cs

```diff
@@ -0,0 +1,30 @@
+using Offices.Application.Extensions;
+using Offices.Application.Services;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+
+builder.Services.AddControllers();
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+
+// Configuring container
+RepositoriesExtension.AddRepositories(builder.Services);
```

Comment:
Extension methods should be called like this: 
`builder.Services.AddRepositories`
And not using the class that contains it

Reply:
Fixed